include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -229,6 +229,12 @@ typedef enum OrtMemType {
   OrtMemTypeDefault = 0,                // the default allocator for execution provider
 } OrtMemType;
 
+// Session Configuration Options
+
+// Key for disable MatMul PrePacking,
+// if the config value is set to "1" then the prepacking is disabled, otherwise prepacking is enabled
```

Comment:
Please comment on the string format for both key and value
And shall we move these into a separated head file or constants.h (add a new sections or namespace for all the supported configs)

Reply:
Seems I would need to move this to a separate header file to avoid build break, or do we want to expose these configs or we can move these into an internal head file?